paasta_tools/cli/cmds/status.py

```diff
@@ -1715,7 +1716,7 @@ def get_replica_state(pod: KubernetesPodV2) -> ReplicaState:
             state = ReplicaState.UNKNOWN
 
         # Catch
-    elif phase == "Failed":
+    elif phase == "Failed" or reason == "Evicted":
```

Comment:
hmm, interesting - evicted pods aren't in the `Failed` phase?

Reply:
oh, I think I see what's happening based on the internal ticket: we're hitting https://github.com/Yelp/paasta/pull/3445/files#diff-36175af498b294330c337132851420b6d63c7469e72162fb6a4496fcd5cbaff2R1677, so I don't think that this change will actually fix the issue since we'll never get to this branch